tests/data/translate_to_txt.py

```diff
@@ -0,0 +1,27 @@
+"""Test transform."""
+
+
+# --- import --------------------------------------------------------------------------------------
+
+
+import pathlib
+import pytest
+
+import WrightTools as wt
+from WrightTools import datasets
+
+# --- tests ---------------------------------------------------------------------------------------
+
+
+@pytest.mark.skip("don't want to write files")
```

Comment:
I would use https://docs.python.org/3/library/tempfile.html?highlight=tempfile#tempfile.NamedTemporaryFile to both clean up after the test and actually test it rather than just skipping